packages/app/translations/en.json

```diff
@@ -49,7 +49,8 @@
         "a11y_change_language": "Select your language",
         "a11y_image_two_boys": "Picture of two people checking their mobile phones",
         "a11y_clear_social_security_input_field": "Clear the social security number field",
-        "a11y_select_login_method": "Select the login method"
+        "a11y_select_login_method": "Select the login method",
+        "button_read_integrity_policy": "We do not collect data about you or your children. Read our privacy policy here"
```

Comment:
För att förkorta lite kan vi skriva: "We collect no data about you or your children. Read our privacy policy here"